hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/schema/Span.java

```diff
@@ -1,10 +1,18 @@
 package org.hypertrace.core.graphql.span.schema;
 
+import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLName;
+import java.util.List;
 import org.hypertrace.core.graphql.common.schema.attributes.AttributeQueryable;
 import org.hypertrace.core.graphql.common.schema.id.Identifiable;
+import org.hypertrace.core.graphql.log.event.schema.LogEvent;
 
 @GraphQLName(Span.TYPE_NAME)
 public interface Span extends AttributeQueryable, Identifiable {
   String TYPE_NAME = "Span";
+  String LOG_EVENT_KEY = "logEvents";
+
+  @GraphQLField
+  @GraphQLName(LOG_EVENT_KEY)
```

Comment:
nonnull

Reply:
This should likely return a result set (may eventually add paging, ordering/filter if needed).

Reply:
> This should likely return a result set

Can that be deferred for now?

Reply:
Any of the impl stuff, yes - but since the response type change would be a breaking api change later, I'd rather make that now (should be trivial unless I'm misunderstanding)

Reply:
> nonnull
Getting this exception after annotating as nonnull 
```
"The field at path '/spans/results[0]/logEvents' was declared as a non null type, but the code involved in retrieving data has wrongly returned a null value.  The graphql specification requires that the parent field be set to null, or if that is non nullable that it bubble up null to its parent and so on. The non-nullable type is '[LogEvent]' within parent type 'Span'",
```

> I'd rather make that now

Ok, I'll update

Reply:
> Getting this exception after annotating as nonnull
> 
> ```
> "The field at path '/spans/results[0]/logEvents' was declared as a non null type, but the code involved in retrieving data has wrongly returned a null value.  The graphql specification requires that the parent field be set to null, or if that is non nullable that it bubble up null to its parent and so on. The non-nullable type is '[LogEvent]' within parent type 'Span'",
> ```

The code returning the list (or result set, with the other comment) should return an empty response rather than null if no value present. So taking the current impl, that may look like updating `get` to `getOrDefault`:
```
    @Override
    public List<LogEvent> logEvents() {
      return spanIdToLogEvents.getOrDefault(id, Collections.emptyList());
    }
```

Reply:
null issue is fixed, but seeing some validation error after moving to result set 
```
"Validation error of type FieldUndefined: Field 'attribute' in type 'LogEventResultSet' is undefined @ 'spans/results/logEvents/attribute'
```
(will try debugging this tomorrow)

Reply:
access pattern will change with the move to result set (which is the breaking change I was referring to): 
```graphql
      logEvents {
        results {
          spanId: attribute(key: "spanId")
          attribute: attribute(key: "attributes")
        }
      }
```

Reply:
done